mesheryctl/pkg/utils/error.go

```diff
@@ -340,6 +343,30 @@ func formatError(msg string, cmd cmdType) string {
 	}
 }
 
+func ErrExtractFile(err error, msg string) error {
+	return errors.New(ErrExtractFileCode, errors.Alert,
+		[]string{"Failed to extract " + msg + " file"},
+		[]string{err.Error()},
+		[]string{"Error occurred while extracting the file"},
+		[]string{"Check the file path and ensure it is a valid " + msg + " file."})
+}
+
+func ErrDirRead(err error, path string) error {
+	return errors.New(ErrDirReadCode, errors.Alert,
+		[]string{"Directory read error. Path: " + path},
+		[]string{err.Error()},
+		[]string{"The provided directory is not present or has an invalid path."},
+		[]string{"To proceed, provide a valid directory path."})
+}
+
+func ErrImportingModel(err error) error {
+	return errors.New(ErrImportingModelCode, errors.Alert,
+		[]string{"Error importing model"},
+		[]string{err.Error()},
+		[]string{"Error occurred while importing the model"},
+		[]string{"Check the model file path and ensure it is a valid model file."})
```

Comment:
What is a valid model file? How does the user verify its validity?

Are we only supporting import of a single file? If not, why does this error only refer to a "model file"?